common/compositor/va/varenderer.cpp

```diff
@@ -132,6 +138,34 @@ bool VARenderer::SetVAProcFilterColorValue(HWCColorControl mode, float value) {
     return false;
   }
 }
+
+bool VARenderer::SetVAProcFilterDeinterlaceValue(HWCDeinterlaceControl mode) {
```

Comment:
this function called for every frame, so you set update_caps_ = true for every frame, then you need to create new deinterlace_ for every frame. this is not necessary.  could we just create/chang deinterlace_ when the value changed? I mean share same deinterlace_ for multiple frames

Reply:
I saw the same issue on color balance too